README.md

```diff
@@ -549,6 +550,22 @@ properly set the `--shm-size` flag depending on the size of your inputs and
 outputs. The default value for docker run command is `64MB` which is very
 small.
 
+## Python Global Interpreter Lock (GIL)
```

Comment:
"Multiple Model Instance Support"?
Then we can talk about how GIL prevents us from attaining concurrency in single python process.